09LessonNine/in-class-project/services.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>We Craft Your Garden</title>
+    <!-- <link rel="stylesheet" href="../../css-reset.css"> -->
+    <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+
+    <main class="services-main clearfix-main">
+        <section class="services-aside">
```

Comment:
try adding a bit more vertical space between nested elements. sometimes it helps visualizing your page